### DISCUSSION TITLE
What happened to the compat files?

### DISCUSSION THREAD

**Discussion by celticalliance** [acemod/ACE3#9527] [General]
Hello,

I have just updated ACE and CBA to 3.16.0, and for some reason the server icons now appear red in the server browser, while I have everything installed correctly on server and client-level. And never had any issues with the previous versions.

I can connect to our servers without issues. I however wonder where the compat files have gone, since we are using RKSL Attachments, and always used the compat_rksl_pm_ii file. 

Can someone please explain to me what I am missing, and how to obtain it? I can not afford for our servers to appear with red icons as we have an Operation on the way. Are the server keys provided with the install correct? There used to be a random number behind it but not anymore.

Thank you for your help.

**Comment by PabstMirror** [comment]
> Yes, the compats we provide still work the same as before, but will now automatically enable and disable themselves. You don't need to enable them in your launcher and can remove them safely.
This includes the following compats: any CDLC, RHS, RH, RKSL, SMA (more might follow in the future).

**Comment by celticalliance** [comment]
But I do not have these compats installed on client-level, nor server-level. We had it in a separate folder, and have removed it. So that can't be it. 

I made a mistake in this post however, ACE's icon in the list of mods shown in the lower-right corner of the server browser appears red. That makes me think there is something missing, that I need to install additionally?

**Comment by PabstMirror** [comment]
compats are now just a part of the main mod
the compats on the workshop were deleted
so you'll want to make sure to cleanup any old `@ace_compat_rhs` mods the server might still have

**Reply by celticalliance** [reply]
Once again, I do not have any compat files installed. Not on client-level, nor on server level. I have those removed.

I just wonder why, after I click on our server in the in-game server browser, ACE's icon in the lower-right corner of the server mod list appears red, while it never did before. 

Is it a server key issue? Do I need to remove certain folders from the ACE folder on server- and client-level?

I have to add though I have no issues joining the server. I'm just responsible for making sure they run smoothly, and don't understand what is missing, or what else I need to do.
